vassal-app/src/main/java/VASSAL/build/AutoConfigurable.java

```diff
@@ -73,7 +73,7 @@
    */
   public VisibilityCondition getAttributeVisibility(String name);
 
-  public static class Util {
+  public class Util {
```

Comment:
Won't this change make the inner class depend on the outer class when it doesn't need to? Is that binary compatible?

Reply:
There is no outer class, it's an interface, completely different thing. Classes inside interfaces are by default static, they cannot be non-static. This is binary compatible, this is not even a change, just removal of redundant information.

Reply:
Ah. I didn't have enough context above this to see that the containing block was an interface.

Reply:
Looks like all the others are also that way.

Reply:
Yes I'm too missing a feature that lets me open the whole file from the PR view, I need it often and it's very inconvenient to open the PR and the changed file locally or navigate to it's version on GitHub in another browser tab.